source/server/http/admin.cc

```diff
@@ -517,39 +517,52 @@ AdminImpl::statsAsJson(const std::map<std::string, uint64_t>& all_stats,
     stats_array.PushBack(stat_obj, allocator);
   }
 
+  Value histograms_container_obj;
+  histograms_container_obj.SetObject();
+
+  Value histograms_obj;
+  histograms_obj.SetObject();
+
+  Stats::HistogramStatisticsImpl empty_statistics;
```

Comment:
It's worth noting in a comment here that an implementation design makes it impossible for the supported quantiles to differ across histograms, although the interface in include/envoy/stats/stats.h, ParentHistogram, makes it look like they can.